src/os/bluestore/BlueStore.cc

```diff
@@ -7786,12 +7784,19 @@ void BlueStore::_kv_sync_thread()
 	  }
 	}
       }
-      if (num_aios) {
```

Comment:
it was an optimization of dubious value (avoid iterating over the list if we know there are no aios), but with the throttle change we have to do the iteration anyway.